api/src/main/resources/messages.properties

```diff
@@ -1,9 +1,9 @@
 ${project.parent.artifactId}.title=HTML Form Entry UI Framework Integration Module
 
 htmlformentryui.viewHtmlForm.breadcrumb = {0}
-htmlformentryui.thisFormIsDeleted = This form has been deleted
+htmlformentryui.thisFormIsDeleted = This form's data has been deleted
```

Comment:
In your screenshot, i do not see any place where the above message is displayed.

Reply:
i attached thescreenshot there